package.json

```diff
@@ -38,6 +39,7 @@
     "immer": "3.1.3",
     "immutable": "^3.8.2",
     "inline-css": "^2.4.1",
+    "klaw-sync": "^6.0.0",
```

Comment:
Hmm, hesitant about addition of these packages but if it was needed then suppose it's fine we have enough already. 

1. Was this functionality not able to be achieved with `fs-extra`?
2. Did the async version not work?

Reply:
1. fs-extra doesn't have a file tree walker, klaw was extracted from it : https://github.com/jprichardson/node-fs-extra#what-happened-to-walk-and-walksync

2. Async probably would work as well, but I didn't see a need for it here.  The code would be a little more convoluted with async, so I was going for comprehensibility over performance.